source/administration/upgrade.md

```diff
@@ -43,7 +43,8 @@ To start, select one of the following guides:
 7. Start your server and address any setting changes relevant in the latest version of Mattermost
       1. Run `sudo start mattermost`.
       2. Opening the **System Console** and saving a change will upgrade your `config.json` schema to the latest version using default values for any new settings added.
-8. Test the system is working by going to the URL of an existing team.
+8. If you have TLS set up on your Mattermost server, run `sudo setcap cap_net_bind_service=+ep ./bin/platform` in your Mattermost directory to allow Mattermost to bind to low ports.
+9. Test the system is working by going to the URL of an existing team.
```

Comment:
Looks good the only thing is that in step 9 I would probably say something like:

Test the ststem is working by going to the URL of the server with a https:// prefix